yarn-project/kv-store/src/indexeddb/multi_map.ts

```diff
@@ -0,0 +1,53 @@
+import type { Key } from '../interfaces/common.js';
+import type { AztecAsyncMultiMap } from '../interfaces/multi_map.js';
+import { IndexedDBAztecMap } from './map.js';
+
+/**
+ * A multi map backed by IndexedDB.
+ */
+export class IndexedDBAztecMultiMap<K extends Key, V>
+  extends IndexedDBAztecMap<K, V>
+  implements AztecAsyncMultiMap<K, V>
```

Comment:
Just a question whether you'd want this to use a different name as compared to the normal map?

Reply:
Mmm, not sure I follow? IDBMultimap extends IDBMap, implements AsyncMultiMap...LGTM even if a tongue twister 🤣